docs/design/operand-deployment-lifecycle-manager.md

```diff
@@ -71,13 +71,13 @@ The Operand (Deployment) Registry Custom Resource (CR) lists OLM Operator inform
 1. `name` of the OperandRegistry
 2. `namespace` of the OperandRegistry
 3. `name` is the name of the operator, which should be the same as the services name in the OperandConfig and OperandRequest.
-4. `namespace` is the namespace where the operator will be deployed.
+4. `namespace` is the namespace where the operator CR will be deployed when InstallMode is `cluster`. Also is the namespace where the operator and operator CR will be deployed when InstallMode is empty or set to `namespace`.
```

Comment:
Could we say "operator's operand" here instead of "operator CR"?  I think given the focus on operands in ODLM it would make more sense to use the term operand here.  It's just a suggestion - feel free to ignore this if you disagree.

Reply:
Actually, I discussed this with @Daniel-Fan offline about this yesterday. My only concern for using "operator's operand" is that ODLM creates the CR for an individual operator, but it doesn't know what is the logic of that operator. Even if the CR for an operator is created in namespace A, its operands could be deployed into other namespaces base on the logic from the operator. Thus we can't say operand is deployed in namespace A.